javascript/CommentsInterface.js

```diff
@@ -149,34 +156,33 @@
 					if(link.hasClass('ham')) {
 						// comment has been marked as not spam
 						comment.html(html);
-						comment.removeClass('spam').hide().fadeIn();
+						comment.removeClass('spam');
 					}
 					else if(link.hasClass('approve')) {
 						// comment has been approved
 						comment.html(html);
-						comment.removeClass('unmoderated').hide().fadeIn();
+						comment.removeClass('unmoderated');
 					}
 					else if(link.hasClass('delete')) {
 						comment.fadeOut(1000, function() {
-							comment.remove();
-									
-							if(commentsList.children().length == 0) {
+                            comment.remove();
+
+							if(commentsList.children().length === 0) {
 								noCommentsYet.show();
 							}
 						});
 					}
 					else if(link.hasClass('spam')) {
-						comment.html(html).addClass('spam').hide().fadeIn();
+						comment.html(html).addClass('spam');
 					}
 				},
 				failure: function(html) {
-					alert(html)
+					alert(html);
```

Comment:
I wonder what is being alerted here? It might be risky to echo out raw HTML to the user.